neurom/core/_soma.py

```diff
@@ -75,15 +81,15 @@ class SomaSinglePoint(Soma):
     Represented by a single point.
     """
 
-    def __init__(self, points):
+    def __init__(self, morphio_soma):
         """Initialize a SomaSinglePoint object."""
-        super().__init__(points)
-        self.radius = points[0][COLS.R]
+        super(SomaSinglePoint, self).__init__(morphio_soma)
```

Comment:
Why do we change from `super().__init__()` to `super(SomaSinglePoint, self).__init__`? Is it a more modern approach?